itou/www/autocomplete/tests.py

```diff
@@ -121,20 +121,17 @@ def test_search_filter_with_rome_code(self):
 class CitiesAutocompleteTest(TestCase):
     def test_autocomplete(self):
 
-        create_test_cities(["67"], num_per_department=10)
+        create_test_cities(["01", "75"], num_per_department=20)
 
         url = reverse("autocomplete:cities")
 
-        response = self.client.get(url, {"term": "alte"})
+        response = self.client.get(url, {"term": "sai"})
         assert response.status_code == 200
-        expected = [
-            {"value": "Altenheim (67)", "slug": "altenheim-67"},
-            {"value": "Altorf (67)", "slug": "altorf-67"},
-            {"value": "Alteckendorf (67)", "slug": "alteckendorf-67"},
-            {"value": "Albé (67)", "slug": "albe-67"},
-            {"value": "Altwiller (67)", "slug": "altwiller-67"},
+        assert json.loads(response.content) == [
```

Comment:
Décode proprement la réponse, et `raise` si le `Content-Type` est incorrect. (plusieurs occurrences)
```suggestion
        assert response.json() == [
```

Reply:
En effet, j'ai naivement repris le code précédent. Je vais faire un tour dans les vues aussi du coup.